### PR TITLE
Fix up amplitude events

### PR DESCRIPTION
**Pull request by mjmayank**
Made some small errors with one field name, and also wanted to optimize where the setID function goes

### CODE REVIEW

File: src/Screens/Onboarding/ActivityType.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ const ActivityType: FunctionComponent<ActivityTypeProps> = ({ navigation }) => {
 
   React.useEffect(() => {
     sendEvent('page_view', {
-      screen: 'onboarding__activity',
+      type: 'onboarding__activity',
```
Comment (mjmayank) [inline src/Screens/Onboarding/ActivityType.tsx:74]: named this field wrong and want it to be consistent with what i did on web

File: src/redux/slices/user/index.ts [hunk 2]
```diff
@@ -101,7 +101,7 @@ export const registerUser = createAsyncThunk(
     }
 
     await setStorage(
-      null,
+      response.user,
```
Comment (mjmayank) [inline src/redux/slices/user/index.ts:104]: any idea why this was null before?

File: src/redux/slices/user/index.ts [hunk 3]
```diff
@@ -418,6 +417,9 @@ const setStorage = async (
   refresh_token?: string,
 ) => {
   if (user) {
+    if (user.id) {
```
Comment (mjmayank) [inline src/redux/slices/user/index.ts:420]: moved this here so that it's in a more central place

File: src/redux/slices/user/index.ts (hunk 2 above)
Comment (Nnanyielugo) [reply src/redux/slices/user/index.ts:104]: At the time, the register endpoint did not return a user object.

**Review by Nnanyielugo — APPROVED** [review]

**Pull request merged**